src/components/QRCode.tsx

```diff
@@ -13,15 +20,26 @@ interface QRCodeProps {
 }
 
 const qrCodeContainer = classnames(
+  position('relative'),
   borderRadius('rounded-2xl'),
   overflow('overflow-hidden'),
   minWidth('min-w-fit')
 )
+const qrCodeLogoContainer = classnames(
+  position('absolute'),
+  display('flex'),
+  alignItems('items-center'),
+  justifyContent('justify-center'),
+  inset('inset-0')
+)
+const qrCodeLogo = classnames(width('w-8'), backgroundColor('bg-primary-dark'))
 
 const QRCodeOptions: Options = {
   type: 'canvas',
   margin: 10,
-  image: '/img/logo.svg',
+  width: 200,
+  height: 200,
+  image: '',
```

Comment:
What if we don't specify `image` at all?

Reply:
Nothing, if we didn't specify an image, it won't generate logo in the center of the QR